client/my-sites/domains/domain-management/dns/index.jsx

```diff
@@ -79,13 +79,6 @@ class Dns extends Component {
 		);
 	}
 
-	renderHeader() {
-		const { translate, selectedDomainName } = this.props;
-		<Header onClick={ this.goBack } selectedDomainName={ selectedDomainName }>
-			{ translate( 'DNS Records' ) }
-		</Header>;
```

Comment:
A `return` was missing here. 🙂

However, I decided to move this code back to the `renderMain` method because #57604 added feature toggling at the controller level, so this whole component (`Dns`) will only be rendered if the `domains/dns-records-redesign` flag is disabled, making the feature flag check here redundant.

Reply:
Makes sense.